src/Microsoft.PowerShell.Commands.Utility/resources/TestJsonCmdletStrings.resx

```diff
@@ -129,4 +129,7 @@
   <data name="JsonSchemaFileOpenFailure" xml:space="preserve">
     <value>Can not open JSON schema file: {0}</value>
   </data>
-</root>
+  <data name="JsonFileOpenFailure" xml:space="preserve">
+    <value>Can not open JSON file: {0}</value>
+  </data>
+</root>
```

Comment:
Ah, also; all files should end with a new line (github shows it with the red symbol, but it just means it needs a newline at the end) so we don't accidentally break any tooling on Linux/MacOS.

Reply:
@vexx32 - I only used Visual Studio 2019 to edit the .resx file. It presents a spreadsheet-like form to enter strings. What did I do to add whitespace and change 'https' to 'http'?

I have edited the .resx file with VS code and I think made the changes. It is checked in.

Reply:
Hmm, that's some of them, yeah. There are a handful of lines that it seems like VS added trailing whitespace to (if you look at the diff on Github in the Files tab from this PR), you can see those lines in green with the added space in a darker green at the end of the line.

Really weird that it would add so many characters that don't need to be there... O.o